sk/src/lib/components/entries/query.ts

```diff
@@ -26,39 +26,78 @@ const SORT_ID_MAP: { [key: string]: string } = {
   format: 'FORMAT'
 }
 
+const SEADEX_SORTERS_LIST: string[] = ["updated"]
+
 // these loads have race conditions, oh well
 
 async function load (pageIndex: number, perPage: number, filterValues: Record<string, unknown>, sortKeys: SortKey[], ids?: number[]) {
-  let sort = SORT_ID_MAP[sortKeys[0]?.id] || undefined
```

Comment:
uuuuuh what happened in these changes? why? so much code, so many changes, idk wtf they are about

Reply:
We wanted to be able to sort via the Pocketbase data not just the Anilist data. So we can now sort by both.